### ISSUE TITLE
[MPM]  Missing  pyevtk in release 9.4.2

### ISSUE THREAD

**Issue by jginternational** [KratosMultiphysics/Kratos#11691]
**Description**
When running any example, if VTK output is enabled, this error is shown: 

```
  File "C:\Program Files\GiD\GiD 16.1.6d\scripts\tohil\python\lib\site-packages\KratosMultiphysics\ParticleMechanicsApplication\particle_vtk_output_process.py", line 2, in <module>
    from pyevtk import hl
ModuleNotFoundError: No module named 'pyevtk'
```

**Scope**
Release 9.4.2

@KratosMultiphysics/mpm

**Comment by ncrescenzio** [comment]
We are aware of this issue. The dependency on `pyevtk` will be removed once #11140 will be merged.

**Comment by roigcarlo** [comment]
Just to comment that we are able to add dependencies now. If you want to keep the module just add it to https://github.com/KratosMultiphysics/Kratos/blob/master/applications/ParticleMechanicsApplication/ParticleMechanicsApplication.json in the dependencies section.

**Comment by jginternational** [comment]
Duplicated #10907

**Comment by ncrescenzio** [comment]
Issue fixed by #11140
